tests/10-rdfc-create.js

```diff
@@ -30,188 +30,189 @@ const {match} = endpoints.filterByTag({
+                expect(
+                  proofs.map(proof => proof?.cryptosuite),
+                  'Expected at least one proof to have ' +
+                '"cryptosuite" property "ecdsa-rdfc-2019" or "ecdsa-jcs-2019".')
+                  .to.contain.oneOf(cryptosuite);
               });
-            /*
-             @link https://w3c.github.io/vc-di-ecdsa/
-             #verify-derived-proof-ecdsa-sd-2023:~:text=
-             The%20cryptosuite%20property%20MUST%20be%20ecdsa%2Drdfc%2D2019%
-             2C%20ecdsa%2Djcs%2D2019%2C%20or%20ecdsa%2Dsd%2D2023.
-             */
-            it('The cryptosuite property of the proof MUST be ' +
-            'ecdsa-rdfc-2019 or ecdsa-jcs-2019.', function() {
-              this.test.cell = {
-                columnId: `${name}: ${keyType}`, rowId: this.test.title
-              };
-              const cryptosuite = ['ecdsa-rdfc-2019', 'ecdsa-jcs-2019'];
-              expect(
-                proofs.map(proof => proof?.cryptosuite),
-                'Expected at least one proof to have ' +
-              '"cryptosuite" property "ecdsa-rdfc-2019" or "ecdsa-jcs-2019".')
-                .to.contain.oneOf(cryptosuite);
-            });
-            it('the signature value (proofValue) MUST be expressed according ' +
-            'to section 7 of [RFC4754] (sometimes referred to as the IEEE ' +
-            'P1363 format) and encoded according to the specific cryptosuite ' +
-            'proof generation algorithm.', async function() {
-              this.test.cell = {
-                columnId: `${name}: ${keyType}`, rowId: this.test.title
-              };
-              const _proof = proofs.find(p =>
-                p?.cryptosuite === 'ecdsa-rdfc-2019');
-              expect(
-                _proof,
-                `Expected VC from issuer ${name} to have an ' +
-                '"ecdsa-rdfc-2019" proof`).to.exist;
-              expect(
-                _proof.proofValue,
-                `Expected VC from issuer ${name} to have a ' +
-                '"proof.proofValue"`
-              ).to.exist;
-              expect(
-                _proof.proofValue,
-                `Expected VC "proof.proofValue" from issuer ${name} to be ` +
-                'a string.'
-              ).to.be.a.string;
-              // now test the encoding which is bs58 for this suite
-              expect(
-                shouldBeBs58(_proof.proofValue),
-                'Expected "proof.proofValue" to be bs58 encoded.'
-              ).to.be.true;
-              // proofBytes will be exactly 64 bytes in size for a P-256 key,
-              // and 96 bytes in size for a P-384 key.
-              const expectedLength = proofLengths[keyType];
-              await shouldHaveByteLength(_proof.proofValue, expectedLength);
-            });
-            it('The "proof" MUST verify with a conformant verifier.',
+              it('the signature value (proofValue) MUST be expressed ' +
```

Comment:
if the `@link` is missing this will need a `@link` to, but again do not address this in current PR.